backend/svc/svcproc.go

```diff
@@ -0,0 +1,119 @@
+package svc
```

Comment:
Self reminder after offline discussion:

we can keep this file here for now, but I don't think it's appropriate to put it in the "backend" directory. It's not AK backend code.

The only caller is the system test - so that's a more logical place, but it's not unreasonable that some AK users may want to use this too as a wrapper for running AK.

`internal` is also not a great choice for the same reason.